Wabbajack.Common/Paths.cs

```diff
@@ -378,23 +380,24 @@ public async Task CopyDirectoryToAsync(AbsolutePath destination)
 
     public struct RelativePath : IPath, IEquatable<RelativePath>, IComparable<RelativePath>
     {
-        private readonly string _path;
+        private readonly string? _nullable_path;
```

Comment:
**Structs are dangerous to have non-nullable members**
Structs are allowed to have non-nullable members.  However, there's a danger here that's not caught by the compiler (that i've seen).
The code will still block you from taking actions or assigning things poorly in the code.. but there's a use case that isn't caught:
A `default` struct will have its non nullable members set to null.  A user will then access the member without checking (as its marked not nullable), and then get a null exception.

It's my understanding so far that structs shouldn't have nullable refs, due to them being able to hold null values if `default` is used.

So instead.. we store a nullable path, and have an internal member that checks and returns empty string in that case.
Thus a `default(RelativePath)` will not fail, despite its internal string member containing null.

Nullability concepts are pretty new.. so my understanding of this issue might evolve and/or they may address this later.